pkg/build/graph/analysis/bc.go

```diff
@@ -238,6 +238,22 @@ func FindPendingTags(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// getImageStreamTagSuggestion will return the appropriate marker Suggestion for when a BuildConfig is missing its input ImageStreamTag;  in particular,
+// it will determine whether or not another BuildConfig can produce the aforementioned ImageStreamTag
+func getImageStreamTagSuggestion(g osgraph.Graph, f osgraph.Namer, tagNode *imagegraph.ImageStreamTagNode, bcNode *buildgraph.BuildConfigNode) osgraph.Suggestion {
+	bcs := []string{}
+	for _, bcNode := range g.PredecessorNodesByEdgeKind(tagNode, buildedges.BuildOutputEdgeKind) {
+		bcs = append(bcs, f.ResourceName(bcNode))
+	}
+	if len(bcs) == 1 {
+		return osgraph.Suggestion(fmt.Sprintf("%s needs to be created by executing `oc start-build %s`.", f.ResourceName(tagNode), strings.Join(bcs[:], ",")))
```

Comment:
See https://github.com/openshift/origin/blob/e7eaff831482b24099caa9aa02e67db5290a442f/pkg/build/graph/analysis/bc.go#L203-l204. Actually, only the command in the case of 1 bc is a suggestion, the rest should be part of the message. So I think getImageSteamTagSuggestion should be getImageStreamTagMarker and return a complete marker instead of just the suggestion.